contributing/code/bugs.rst

```diff
@@ -32,9 +33,19 @@ If your problem definitely looks like a bug, report it using the official bug
 * Give as much detail as possible about your environment (OS, PHP version,
   Symfony version, enabled extensions, ...);
 
+* If you want to provide a stack trace you got on an html page, be sure to
```

Comment:
I think I'd recommend the opposite: always share the stack traces as images because they offer some protection against Google, etc. Remember that traces can contain sensitive information, so we don't make them searchable on Internet.